src/tfr_description/xacro/proto_drivebase_controller.xacro

```diff
@@ -0,0 +1,49 @@
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="proto">
+  <!--xacro dectiptio of the prototype skid steer controller-->
+  <xacro:macro name="wheel_joint" params="wheel">
+    <gazebo reference="wheel_${wheel}_joint">
+      <kp value="100000.0"/>
+      <kd value="1.0"/>
+      <minDepth value="0.005"/>
+      <mu1 value="0.45"/> <!-- 2.0 -->
+      <mu2 value="0.4"/> <!-- 0.5 -->
+      <fdir1 value="1 0 0"/>
+      <turnGravityOff>false</turnGravityOff>
+    </gazebo>
+  </xacro:macro>
+  
+  <xacro:wheel_joint wheel="fr" />
+  <xacro:wheel_joint wheel="rr" />
+  <xacro:wheel_joint wheel="fl" />
+  <xacro:wheel_joint wheel="rl" />
+
+<!--  <gazebo>
+    <plugin name="gazebo_ros_control" filename="libgazebo_ros_control.so">
+      <robotNamespace></robotNamespace>
+    </plugin>
+  </gazebo> -->
+  
+  <gazebo>
+    <plugin name="skid_steer_drive_controller" filename="libgazebo_ros_skid_steer_drive.so">
+      <updateRate>50.0</updateRate>
+      <robotNamespace></robotNamespace>
+      <leftFrontJoint>wheel_fl_joint</leftFrontJoint>
+      <rightFrontJoint>wheel_fr_joint</rightFrontJoint>
+      <leftRearJoint>wheel_rl_joint</leftRearJoint>
+      <rightRearJoint>wheel_rr_joint</rightRearJoint>
+      <wheelSeparation>${base_width}</wheelSeparation>
+      <wheelDiameter>${wheel_radius*2}</wheelDiameter>
+      <robotBaseFrame>god</robotBaseFrame>
+      <torque>20</torque>
+      <topicName>cmd_vel</topicName>
+      <commandTopic>cmd_vel</commandTopic>
+      <!-- we are simulating the output of sensor fusion right here -->
```

Comment:
Is this actually producing fused odometry? Or are you just saying that, in the absence of other sensors to do fusion, we're just using this as the fused odometry even though it's only one source?

Reply:
(if that's the case, no worries, I'm just clarifying)

Reply:
The reason I did that was to make sure that rtab map was subscribed to the correct topic, the idea was that we would be able to swap the gazebo simulation with the real thing with no issues.

Reply:
Got it, I figured it was something like that.